web-ext-config.js

```diff
@@ -15,7 +15,6 @@ module.exports = {
   },
   ignoreFiles: [
     "bin",
-    "core-addon",
```

Comment:
@rhelmer do we really only want the `FirefoxPrivileged*` files from that directory? In any case, that's fine too. Sorry for breaking it!